src/plugins/orangehrmAuthenticationPlugin/Subscriber/AuthenticationSubscriber.php

```diff
@@ -80,7 +81,10 @@ public function onControllerEvent(ControllerEvent $event): void
             return;
         }
 
-        if ($this->getControllerInstance($event) instanceof AbstractViewController) {
+        if (
+            $this->getControllerInstance($event) instanceof AbstractViewController ||
+            $this->getControllerInstance($event) instanceof AbstractModuleController
```

Comment:
if you get timed out when clicking on a module name specifically the redirect URL would not get set to `AuthUser`. this should fix that.